tests/integration_tests/embeddings/test_huggingface.py

```diff
@@ -21,3 +21,18 @@ def test_huggingface_embedding_query() -> None:
     embedding = HuggingFaceEmbeddings()
     output = embedding.embed_query(document)
     assert len(output) == 768
+
+def test_huggingface_instructor_embedding_documents() -> None:
+    """Test huggingface embeddings."""
+    documents = ["foo bar"]
+    embedding = HuggingFaceEmbeddings(model_name="hkunlp/instructor-large", instruction="Represent the text")
+    output = embedding.embed_documents(documents)
+    assert len(output) == 1
+    assert len(output[0]) == 768
+
+def test_huggingface_instructor_embedding_query() -> None:
+    """Test huggingface embeddings."""
+    query = "foo bar"
+    embedding = HuggingFaceEmbeddings(model_name="hkunlp/instructor-large", instruction="Represent the text")
+    output = embedding.embed_query(query)
+    assert len(output[0]) == 768
```

Comment:
I think this needs to match the test on line 23 ie 
```suggestion
    assert len(output) == 768
```
This ensures that the outputs can be used the same by both ST and instructor models (also see comment in query embedding)

Reply:
This is a good catch, fixing.